app/controllers/modules/course/module_host.rb

```diff
@@ -64,9 +78,46 @@ def get_settings_items(controller)
     class_methods_module.module_eval do
       include Sidebar
       include Settings
+      include Enableable
     end
   end
 
   # Eager load all the modules declared.
   eager_load_modules(File.join(__dir__, '../'))
+
+  # Initialize the module host instance
+  #
+  # @param [SettingsObject] instance settings object
```

Comment:
OK, But sometimes duck typing will not have specific types, how should we describe the type in that case?